website/src/pages/docs/features/jwt.mdx

```diff
@@ -170,6 +170,25 @@ const yoga = createYoga({
 })
 ```
 
+#### WebSockets Connection Params
+
+You can configure the plugin to look for the token in the connection params of a WebSocket
+connection if you use `graphql-ws`.
+
+```ts
+import { extractFromConnectionParams, useJWT } from '@graphql-yoga/plugin-jwt'
+
+const yoga = createYoga({
+  // ...
+  plugins: [
+    useJWT({
+      // So it will look for the token in the connectionParams.my-token field in case of a WebSockets connection
+      lookupLocations: [extractFromConnectionParams({ name: 'my-token' })]
```

Comment:
@ardatan can we maybe add here an example of how the WS request should look like? Because I did not understand how the end-user passes the `connectionParams` to the plugin?

Reply:
Connection params are not passed by the user explicitly. It happens on GraphQL WS integration level.

Reply:
I added a reference to GraphQL WS integration docs

Reply:
I see, but how does the user pass that to the server? I suggest to add an flow example/doc here that explains how the user passes the token, to the WS and then to the JWT plugin?

Reply:
I added an example with GraphQL WS client.